tools/hermetic_grunt/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Clean the tscache while we figure out how to make it work across calls.
+# TODO(fortuna): Fix this.
+rm -rf $REPOSITORY_ROOT/out/.tscache &&
+
+# We can't link the Gruntfile because grunt resolves the link before using its
+# directory as the base to run.
+cp $REPOSITORY_ROOT/Gruntfile.coffee . &&
+mkdir -p $REPOSITORY_ROOT/out/.tscache &&
+ln -s $REPOSITORY_ROOT/{version.py,.git,src,out/.tscache} . &&
+
+grunt "$@" ||
+
+# Copy build output to the out/ directory.
+# TODO(fortuna): Make grunt output to out/ directly.
+rm --recursive build/{third_party,tools} &&
```

Comment:
Several lines here have a traililng `&&` but either have no subsequent command or end with a newline, with no `\` - are you sure they do what you want? I would recommend putting `set -e` near the top of the file so that `bash` will terminate execution on the first error.